udata/core/topic/forms.py

```diff
@@ -2,9 +2,30 @@
 from udata.forms import ModelForm, fields, validators
 from udata.i18n import lazy_gettext as _
 
-from .models import Topic
+from .models import Topic, TopicElement
 
-__all__ = ("TopicForm",)
+__all__ = ("TopicForm", "TopicElementForm")
+
+
+class TopicElementForm(ModelForm):
+    model_class = TopicElement
+
+    title = fields.StringField(_("Title"))
+    description = fields.StringField(_("Description"))
+    tags = fields.TagField(_("Tags"))
+    extras = fields.ExtrasField()
+    element = fields.ModelField(_("Element"))
+
+    def validate(self, extra_validators=None):
+        """
+        Make sure that either title or element is set.
+        (Empty nested element is a valid use case for "placeholder" elements)
+        """
+        validation = super().validate(extra_validators)
```

Comment:
I have an issue with `csrf_token`missing, that I think is coming from this validation (either with curl or swaggerUI).

Reply:
Could you reproduce the issue?